src/app/components/Item/ItemHoldings.jsx

```diff
@@ -75,53 +76,8 @@ class ItemHoldings extends React.Component {
     const bibId = this.props.bibId;
 
     return (
-      <table className="nypl-basic-table">
-        <caption className="hidden">Item details</caption>
-        <thead>
-          <tr>
-            <th>Location</th>
-            <th>Call No.</th>
-            <th>Status</th>
-            <th>Message</th>
-            <th></th>
-          </tr>
-        </thead>
-        <tbody>
-          {
-            itemsToDisplay.map((h, i) => {
-              let itemLink;
-              let itemDisplay = null;
-
-              if (h.requestHold) {
-                itemLink = h.availability === 'available' ?
-                  <Link
-                    className="button"
-                    to={`/hold/request/${bibId}-${h.id}`}
-                    onClick={(e) => this.getRecord(e, bibId, h.id)}
-                  >Request</Link> :
-                  <span className="nypl-item-unavailable">Unavailable</span>;
-              }
-
-              if (h.callNumber) {
-                itemDisplay =
-                  <span dangerouslySetInnerHTML={this.createMarkup(h.callNumber)}></span>;
-              } else if (h.isElectronicResource) {
-                itemDisplay = <span>{h.location}</span>;
-              }
-
-              return (
-                <tr key={i} className={h.availability}>
-                  <td>{h.location}</td>
-                  <td>{itemDisplay}</td>
-                  <td>{h.status}</td>
-                  <td>{h.accessMessage}</td>
-                  <td>{itemLink}</td>
-                </tr>
-              );
-            })
-          }
-        </tbody>
-      </table>
+      (itemsToDisplay && _isArray(itemsToDisplay) && itemsToDisplay.length > 0) ?
```

Comment:
You can just do `itemsToDisplay.length` instead of `> 0`. It's the same thing since 0 is falsy and any number greater than that is truthy.